src/main/flight/pid.h

```diff
@@ -85,11 +85,6 @@ typedef struct pidProfile_s {
     float rateAccelLimit;                   // accel limiter roll/pitch deg/sec/ms
 } pidProfile_t;
 
-#if FLASH_SIZE <= 128
```

Comment:
This is an unrelated change. If we want to do this, it should be in a separate PR